vcxagency-node/src/scripts/download-certs.js

```diff
@@ -0,0 +1,58 @@
+const { S3Client, GetObjectCommand } = require('@aws-sdk/client-s3')
+const fs = require('fs')
+const logger = require('../logging/logger-builder')(__filename)
+
+function parseAssetPath (path) {
```

Comment:
Sorry for nagging about this again, but we could avoid all this parsing and error handling if we'd just ask user to specify S3 path to cert and key :-) I agreed with the parsing as you suggested, but only now when it's written out I see couple issues - which can be fixed, but there would be less code to write and probably better error messages.

The few small problems I see:
- If user enters `bucket` instead of `bucket/foo` it will tell him that bucket key is missing. That message is incorrect because `foo` itself is not actual bucket key, but something we use to construct bucket key - like `foo.crt`.
- `foo/` is currently considered valid value 
-  can't store certificate on deeper paths like `bucket/foo/bar.crt`

Reply:
I'd suggest 3 params
`AWS_S3_BUCKET_CERTS`
`AWS_S3_PATH_CERT`
`AWS_S3_PATH_CERT_KEY`

Then there's no need for parsing, just pass the stuff to S3 client and see if it throw some error or not :-)